astroplan/core.py

```diff
@@ -37,70 +54,94 @@ def __init__(self, name, longitude, latitude, elevation, pressure,
+        if location is None and (latitude is not None and longitude is not None):
+            accepted_latlon_types = [str, Quantity]
+            if (type(latitude) in accepted_latlon_types and
+                type(longitude) in accepted_latlon_types):
+                latitude = Latitude(latitude, unit=u.degree)
+                longitude = Longitude(longitude, unit=u.degree)
 
-        Parameters
-        ----------
-        pressure : `~astropy.units.Quantity`
-            The ambient pressure.
+            self.location = EarthLocation.from_geodetic(longitude, latitude,
+                                                        elevation)
 
-        relative_humidity : `~astropy.units.Quantity`
-            The ambient relative humidity.
+        elif isinstance(location, EarthLocation):
+            self.location = location
 
-        temperature :
-            The ambient temperature.
-        """
-        raise NotImplementedError()
+        else:
+            raise ValueError, ('Observatory location must be specified with '
+                    'either (1) latitude and longitude in degrees as '
+                    'accepted by astropy.coordinates.Latitude and '
+                    'astropy.coordinates.Latitude, or (2) as an instance of '
+                    'astropy.coordinates.EarthLocation.')
 
-    def get_date(self, input_date_time, timezone=None):
-        """
-        Builds an object containing date and time information.
+        # Accept various timezone inputs, default to UTC
+        if isinstance(timezone, datetime.tzinfo):
+            self.timezone = timezone
+        elif type(timezone) == str:
+            self.timezone = pytz.timezone(timezone)
+        else:
+            raise ValueError, ('timezone keyword should be a string, or an '
+                               'instance of datetime.tzinfo')
 
-        Default time zone is UTC.
-        If time zone (local or otherwise) requested, then uses `datetime.datetime`
-        and pytz.timezone to convert from UTC.
 
-        Must first create date_time object with get_date() before using any other 
-        Observer methods, all of which require one as an argument.
+    def altaz(self, target, time):
+        """
+        Returns an instance of `~astropy.coordinates.SkyCoord` with altitude and
+        azimuth of the `FixedTarget` called `target` at time `time`.
 
         Parameters
         ----------
-        input_date_time : WHAT TYPE IS astropy.time OBJECT?
+        target : `~astroplan.FixedObject`
+            Celestial object of interest.
 
+        time : `~astropy.time.Time`
+            Object
         """
-        raise NotImplementedError()
+        if not isinstance(target, FixedTarget):
+            raise TypeError, ('The target must be an instance of FixedTarget')
+
+        return target.coord.transform_to(AltAz(target.coord.ra, target.coord.dec,
+                                         location=self.location, obstime=time))
```

Comment:
Here I assume the `target` input to `obs.altaz(target, time)` is an instance of `FixedTarget`, but it could be a moving target in the future, or we may choose to support SkyCoords. Any opinions on interchangeability with SkyCoords where feasible?